daemon/logger/gelf/gelf.go

```diff
@@ -85,12 +85,13 @@ func New(info logger.Info) (logger.Logger, error) {
 	}
 
 	var gelfWriter gelf.Writer
-	if address.Scheme == "udp" {
+	switch address.Scheme {
+	case "udp":
 		gelfWriter, err = newGELFUDPWriter(address.Host, info)
 		if err != nil {
 			return nil, err
 		}
-	} else if address.Scheme == "tcp" {
+	case "tcp":
```

Comment:
While we're updating, we should probably add a `default:` with a `// TODO` comment here, to consider returning an error for unsupported schemes.